database/seeders/DatabaseSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     *
+     * @return void
+     */
+    public function run()
```

Comment:
Please comment all lines in the run() function, 2FAuth do not need any seeds when using the `php artisan migrate --seed` command

Reply:
Done :)